.eslintrc.json

```diff
@@ -1,14 +1,11 @@
 {
-  "parser": "@typescript-eslint/parser", // Specifies the ESLint parser
   "parserOptions": {
     "ecmaVersion": 2020, // Allows for the parsing of modern ECMAScript features
     "sourceType": "module" // Allows for the use of imports
   },
   "extends": [
     "plugin:@typescript-eslint/recommended", // Uses the recommended rules from the @typescript-eslint/eslint-plugin
-    "prettier/@typescript-eslint", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
```

Comment:
why is this change?

Reply:
`eslint-config-prettier` was updated in this PR and this version includes it:
`
"prettier/@typescript-eslint" has been merged into "prettier" in eslint-config-prettier 8.0.0
`